optimum/utils/testing_utils.py

```diff
@@ -4,6 +4,34 @@
 import sys
 import unittest
 
+from packaging import version
+
+
+def is_accelerate_available():
```

Comment:
duplicate with `[optimum/bettertransformer/utils.py](https://github.com/huggingface/optimum/pull/423/files#diff-6df0494a0af916fb9a243bd6bea72f13cb284ccbdbccfdab2a12241828c5d037)`

Maybe we can put it in `optimum/utils/__init__.py` as there are already so available checks there

Reply:
Thanks! Refactored in e8e34729c95fc4553c3b9158802e0b036c4b881f